packages/core/__tests__/screens/__snapshots__/W3cProofRequest.test.tsx.snap

```diff
@@ -1044,13 +1044,13 @@ exports[`displays a proof request screen ProofRequest Screen, W3C displays a pro
                             style={
                               Array [
                                 Object {
-                                  "backgroundColor": "#5f0f7b",
+                                  "backgroundColor": "#781e4b",
```

Comment:
Q: What is causing this background colour to change?

Reply:
> 7 Months Ago (April 2025):
> Commit b2987d7b: Filipe updated the OCA Branding code
> The actual running code changed to use #781e4b
> BUT the test snapshot file wasn't fully updated - it still had some old #5f0f7b values

> Yesterday (Your Commit ec60e0d0):
> You made changes to credential naming logic
> You ran the tests
> Jest noticed: "Hey, the snapshot says #5f0f7b but the actual output is #781e4b"
> Jest updated the snapshot to match what the code actually produces

Reply:
does this make sense to you?